index.bs

```diff
@@ -1334,9 +1335,10 @@ in order to reduce resource consumption, notably battery usage.
     : output
     :: None
 
-    1.  Set |sensor_instance|.{{[[state]]}} to "idle".
-    1.  [=Fire an event=] named "error" at |sensor_instance| using {{SensorErrorEvent}}
-        with its {{SensorErrorEvent/error!!attribute}} attribute initialized to |error|.
+    1.  [=Queue a task=] to run these steps:
```

Comment:
What about 'unregister' and resetting internal slots? Do we have issue for that?